claat/fetch/inprocess/inprocess.go

```diff
@@ -0,0 +1,22 @@
+package inprocess
+
+import "io"
+
+// InProcessFetcher implements fetch.Fetcher. It retrieves resources from an in-process source via io.Reader.
```

Comment:
nit: line longer than 80 characters

Reply:
80 char limit isn't typically enforced in Go files